### PR TITLE
Feature/docu and iet patterns

### PR DESCRIPTION
**Pull request by manuwei**
* add iet patterns
* fix some latex statements that our new markdown-it-katexx lib can not handle 

Before merging this PR, the IET Paper should appear and a proper citation to the paper should be added. 
To do this, execute the patternatlas via docker with the PATTERN_ATLAS_CONTENT_REPOSITORY_BRANCH set to this branch and add the paper reference:

step 1: open the pattern settings:
image

step 2: edit the paper ref
image

step 3: perform a database backup with pgadmin (see readme https://github.com/PatternAtlas/pattern-atlas-content) and update db-backup-files/2-patternatlas-data-inserts.sql accordingly

step 4: test and merge the PR

### CODE REVIEW

File: db-backup-files/2-patternatlas-data-inserts.sql [hunk 1]
```diff
@@ -263,29 +264,29 @@ INSERT INTO public.pattern VALUES ('678a2e5a-697f-4416-afe6-fd18bc935bdc', 'Hybr
...
...nt a pixel''s color information in the flexible representation of quantum image (FRQI) while the position is represented by an additional register [(Yan, Iliyasu and Venegas-Andraca 2015)](https://link.springer.com/content/pdf/10.1007/s11128-015-1195-6.pdf).\nFor quantum neural networks, a so-called quantum neuron (quron) makes use of this encoding [(Schuld, Sinayskiy and Petruccione 2014)](https://link.springer.com/article/10.1007/s11128-014-0809-8).\nA state preparation routine to create this encoding is provided by  [PennyLane](https://pennylane.readthedocs.io/en/stable/introduction/templates.html) where a rotation on the axis x, y, or z can be chosen. \nSince the state preparation is straightforward (see solution sketch), this encoding can easily be implemented using default qubit rotations [(Leymann and Barzen 2020)](https://iopscience.iop.org/article/10.1088/2058-9565/abae7d).\n\n ", "Related Patterns": "As this pattern provides a concrete encoding strategy, it specifies [Initialization](pattern-languages/af7780d5-1f97-4536-8da7-4194b093ab1d/312bc9d3-26c0-40ae-b90b-56effd136c0d).\n\n"}', 'af7780d5-1f97-4536-8da7-4194b093ab1d', 'https://ieeexplore.ieee.org/document/9425837/');
```
Comment (salmma) [inline db-backup-files/2-patternatlas-data-inserts.sql]: a [ is missing in solution at "taken from ..." such that the link is presented

Comment (salmma) [reply db-backup-files/2-patternatlas-data-inserts.sql]: in addition, in section result, the cite is shown in latex format: "\cite{...}"

File: db-backup-files/2-patternatlas-data-inserts.sql [hunk 2]
```diff
@@ -263,29 +264,29 @@ INSERT INTO public.pattern VALUES ('678a2e5a-697f-4416-afe6-fd18bc935bdc', 'Hybr
...
...f(x)} \\state{x}  \\right) \\otimes \\state{-} $$", "Variants": "Enter your input for this section here.", "Known Uses": "Many algorithms, including the algorithms of Deutsch, Deutsch-Jozsa, Grover, and Shor make use of function tables.", "Related Patterns": "This pattern uses [Initialization](pattern-languages/af7780d5-1f97-4536-8da7-4194b093ab1d/312bc9d3-26c0-40ae-b90b-56effd136c0d) and [Uniform Superposition](pattern-languages/af7780d5-1f97-4536-8da7-4194b093ab1d/2229a430-fe92-4411-9d72-d10dd1d8da14) to prepare the initial state of the quantum register. \nFunction tables can be further generalized as [Amplitude Amplification](pattern-languages/af7780d5-1f97-4536-8da7-4194b093ab1d/96b4d28a-a5ce-4c96-85df-d42587b13c57). \nThe computation of the function table is done by an [Oracle](pattern-languages/af7780d5-1f97-4536-8da7-4194b093ab1d/1cc7e9d6-ab37-412e-8afa-604a25de296e). \nOften, [Uncompute](pattern-languages/af7780d5-1f97-4536-8da7-4194b093ab1d/d4f7c247-e2bb-4301-ad06-f758fa58f2dc) is required to dissolve the created entanglement for subsequent steps of the algorithm."}', 'af7780d5-1f97-4536-8da7-4194b093ab1d', 'https://link.springer.com/chapter/10.1007/978-3-030-14082-3_19');
```
Comment (salmma) [inline db-backup-files/2-patternatlas-data-inserts.sql]: in section Result, first '' is used instead of "

Comment (salmma) [reply db-backup-files/2-patternatlas-data-inserts.sql]: further, in Relations to other Patterns the isRelatedTo is presenting strange rendering--> "Function Table: Applying U_{f}U f ​ Applying U_{f}U f ​ Applying the unitary ..."
same case also in pattern Creating Entanglement

File: db-backup-files/2-patternatlas-data-inserts.sql [hunk 3]
```diff
@@ -263,29 +264,29 @@ INSERT INTO public.pattern VALUES ('678a2e5a-697f-4416-afe6-fd18bc935bdc', 'Hybr
...
...2039679/). \nConsequently, the address and output registers can be in  superposition instead of only single values.\nThe solution sketch demonstrates the loading process of a QRAM [(Giovannetti, Lloyd and Maccone 2008)](https://arxiv.org/abs/0708.1879) holding a superposition of the first two addresses ($\\newcommand{\\state}[1]{{\\left| #1 \\right>}} \\frac{1}{\\sqrt{2}}\\state{00} + \\frac{1}{\\sqrt{2}}\\state{01})$ as input register. \nLoading the data values of the corresponding addresses results in the following state:\n\n\n$$\n\\newcommand{\\colVec}[1]{% inline column vector\n\\bigl( \\begin{smallmatrix}#1\\end{smallmatrix}\\bigr)\n}\n\\newcommand{\\bigColVec}[1]{% inline column vector\n\\left( \\begin{matrix}#1\\end{matrix}\\right)\n}\n\\newcommand{\\state}[1]{{\\left| #1 \\right>}}\\state{\\psi} = \\frac{1}{\\sqrt{2}} \\state{00}\\state{010} + \\frac{1}{\\sqrt{2}}\\state{01}\\state{110}\n$$  \n\n\nLoading $m$ of $n$ data values using a QRAM can be generalized as follows [(Schuld and Petruccione 2018)](https://www.springer.com/gp/book/9783319964232): \n\n\n$$\n\\newcommand{\\colVec}[1]{% inline column vector\n\\bigl( \\begin{smallmatrix}#1\\end{smallmatrix}\\bigr)\n}\n\\newc
```
Comment (salmma) [inline db-backup-files/2-patternatlas-data-inserts.sql]: same as for Angle Encoding: under solution the last link is not presented correctly as a [ is missing

Comment (salmma) [inline db-backup-files/2-patternatlas-data-inserts.sql:284]: in the paper it is stated that QPE uses Matrix Encoding and on the other hand that Matrix Encoding can be used to define a unitary operation as input for QPE. 

Here, currently, Matrix Encoding canBeUsedWith QPE is defined in the content. @lharzenetter what do you think?

Comment (salmma) [inline db-backup-files/2-patternatlas-data-inserts.sql]: in solution: the SummerSOC paper is still "to appear" and has to be updated

**Review by salmma — CHANGES_REQUESTED** [review]
See my comments below.

**Review by lharzenetter — APPROVED** [review]
All comments seem to be resolved!

**Pull request merged**